### ISSUE TITLE
outdated Commit in the Asset Entry

### ISSUE THREAD

**Issue by betalars** [LemiSt24/line-renderer#3]
Original Issue was incorrect, See the comment by @KoBeWi

**Comment by betalars** [comment]
Sorry, I forgot to check the files in this repository. Godot Team seems to have messed up here.

**Comment by KoBeWi** [comment]
No, the addon was wrongly submitted. It didn't provide the newest hash and Godot downloads older version.
You should update your asset entry to use the newest commit.
